src/lib/db/clients/cockroachdb.js

```diff
@@ -0,0 +1,606 @@
+// Copyright (c) 2015 The SQLECTRON Team
```

Comment:
Instead of copying the postgresql client you can just provide a mapping to it.
The server / database code is a little convoluted (sorry), but checkout how I added `MariaDB` in `lib/db/clients/index.js`

You could do the same thing with CockroachDB - add an entry to `CLIENTS`, and add a mapping to the default export.

i think that's the only thing you need

Reply:
Awesome, I should have checked there first!

There are a couple of differences between CockroachDB and PostgresSQL though. CockroachDB doesn't support table triggers, so cockroachdb.js has a no-op implementation of `listTableTriggers`. Also, the way to cancel a query is different in CockroachDB, so I changed that too.

Is there a way you'd recommend I can reduce code duplication, but still account for those differences?